error_test.go

```diff
@@ -16,7 +16,7 @@ func Test_Error(t *testing.T) {
 	err = Mask(testError)
 
 	got := fmt.Sprintf("%#v\n", err)
-	r, err := regexp.Compile(`[.* {{"desc":"","docs":"","kind":"testError"}}]`)
+	r, err := regexp.Compile(`[.* test error]`)
```

Comment:
We didn't have `error` suffixes so far.

Reply:
I think it is better that way. We have `waitError` error types for instance and all it then says is `wait`, which is rather confusing.

Reply:
I think this is because the error name is bad. It should be `timeoutError` or `waitedTooLongError`. Then it isn't confusing at all. `wait error` still looks dump IMO.

Reply:
Removing it magically from the kind feels wrong. The rule is the message is generated from the type. So there is nothing hidden. It is just converted. I would like to keep it that way and see where we go from here in the future. I can imagine we will change more down the road.